src/Sql/Sql.Test/LiveTests/TestLiveScenarios.ps1

```diff
@@ -0,0 +1,99 @@
+    -ServerName $ServerName `
+    -Location $ResourceGroupLocation `
+    -SqlAdministratorCredentials `
+    $(New-Object -TypeName System.Management.Automation.PSCredential -ArgumentList $AdminSqlLogin, $(ConvertTo-SecureString -String $Password -AsPlainText -Force))
+
+Invoke-LiveTestScenario -Name "Create a Sql Database" -Description "Test New-AzSqlDatabase" -NoResourceGroup -ScenarioScript `
+{
+    $dbName = New-LiveTestResourceName
+
+    $actual = New-AzSqlDatabase -ResourceGroupName $RgName -ServerName $ServerName -DatabaseName $dbName
+    $actual.ServerName
+    Assert-AreEqual $ServerName $actual.ServerName
+    Assert-AreEqual $RgName $actual.ResourceGroupName
+    Assert-AreEqual $dbName $actual.DatabaseName
+    Remove-AzSqlDatabase -ResourceGroupName $RgName -ServerName $ServerName -DatabaseName $dbName
+}
+
+Invoke-LiveTestScenario -Name "Get a Sql Database" -Description "Test Get-AzSqlDatabase" -NoResourceGroup -ScenarioScript `
+{
+    $dbName = New-LiveTestResourceName
+
+    $null = New-AzSqlDatabase -ResourceGroupName $RgName -ServerName $ServerName -DatabaseName $dbName
+    $actual = Get-AzSqlDatabase -ResourceGroupName $RgName -ServerName $ServerName -DatabaseName $dbName 
+    Assert-AreEqual $serverName $actual.ServerName
+    Assert-AreEqual $rgName $actual.ResourceGroupName
+    Assert-AreEqual $dbName $actual.DatabaseName
+    Remove-AzSqlDatabase -ResourceGroupName $RgName -ServerName $ServerName -DatabaseName $dbName
+}
+
+Invoke-LiveTestScenario -Name "Update a Sql Database" -Description "Test Set-AzSqlDatabase" -NoResourceGroup -ScenarioScript `
+{
+    $dbName = New-LiveTestResourceName
+
+    $null = New-AzSqlDatabase -ResourceGroupName $RgName -ServerName $ServerName -DatabaseName $dbName
+    $null = Set-AzSqlDatabase -ResourceGroupName $RgName -ServerName $ServerName -DatabaseName $dbName -Edition "Standard" -RequestedServiceObjectiveName "S0"
+    $actual = Get-AzSqlDatabase -ResourceGroupName $RgName -ServerName $ServerName -DatabaseName $dbName
+    Assert-AreEqual $dbName $actual.DatabaseName
+    Assert-AreEqual "S0" $actual.RequestedServiceObjectiveName
+    Assert-AreEqual "Standard" $actual.Edition 
+    Remove-AzSqlDatabase -ResourceGroupName $RgName -ServerName $ServerName -DatabaseName $dbName
+}
+
+Invoke-LiveTestScenario -Name "Remove a Sql Database" -Description "Test Remove-AzSqlDatabase" -NoResourceGroup -ScenarioScript `
+{
+    $dbName = New-LiveTestResourceName
+
+    $null = New-AzSqlDatabase -ResourceGroupName $RgName -ServerName $ServerName -DatabaseName $dbName
+    Remove-AzSqlDatabase -ResourceGroupName $RgName -ServerName $ServerName -DatabaseName $dbName
+    $actual = Get-AzSqlDatabase -ResourceGroupName $RgName -ServerName $ServerName
+
+    Assert-False { $actual.DatabaseName -contains $dbName}
+}
+# At the end of db test, clear the sql server and the resource group
+Remove-AzSqlServer -ResourceGroup $RgName -ServerName $ServerName
```

Comment:
This statement is not necessary. Clear-LiveTestResources is used to remove the resource group which will delete all the resources created underneath it.

Reply:
Updated.